src/bgnbd_PAlive.cpp

```diff
@@ -0,0 +1,152 @@
+#include <RcppArmadillo.h>
+#include <math.h>
+#include "clv_vectorized.h"
+
+
+arma::vec bgnbd_PAlive(const double r,
+                       const arma::vec& vAlpha_i,
+                       const arma::vec& vA_i,
+                       const arma::vec& vB_i,
+                       const arma::vec& vX,
+                       const arma::vec& vT_x,
+                       const arma::vec& vT_cal){
+  arma::vec n_term1 = (vA_i/(vB_i + vX - 1)) % clv::vec_pow((vAlpha_i + vT_cal)/(vAlpha_i + vT_x), (r+vX));
+
+  return (1 / (1 + (vX > 0) % n_term1));
+}
+
+//' @title BG/NBD: PAlive without covariates
+//'
+//' @description
+//' BG/NBD without Covariates: Calculates the probability of a customer being alive
+//' at the end of the calibration period.
+//'
+//' @param r TODO: describe
+//' @param alpha TODO: describe
+//' @param a TODO: describe
+//' @param b TODO: describe
+//' @template template_params_rcppxtxtcal
+//'
+//' @details
+//' \code{r} \code{alpha} \code{a} \code{b} estimated parameters in original scale
+//' for the BG/NBD model, namely (r, alpha, a, b).
+//' r and alpha: TODO: description.
+//' a and b: TODO: description
+//'
+//'
+//'@return
+//'Returns a vector with the PAlive for each customer.
+//'
+//' @name bgnbd_nocov_PAlive
+// [[Rcpp::export]]
+arma::vec bgnbd_nocov_PAlive(const double r,
```

Comment:
move all docu to bgnbd_PAlive and link with @rdname bgnbd_PAlive. remove existing @name tags and only keep @name bgnbd_PAlive